src/createAnimatedComponent.tsx

```diff
@@ -42,6 +50,7 @@ import { getShadowNodeWrapperFromRef } from './reanimated2/fabricUtils';
 import updateProps from './reanimated2/UpdateProps';
 import NativeReanimatedModule from './reanimated2/NativeReanimated';
 import { isSharedValue } from './reanimated2';
+import { AnimateProps } from './reanimated2/helperTypes';
```

Comment:
Why don't we import type here?
Can we add [eslint rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md) to always import type if possible? WDYT?

```suggestion
import type { AnimateProps } from './reanimated2/helperTypes';
```

Reply:
This is a great idea. I have done a test run and got (treating as errors):
![Screenshot 2023-06-29 at 09 31 40](https://github.com/software-mansion/react-native-reanimated/assets/40713406/58d5d4fe-2e1b-46ff-b263-f2ac9b61bdc8)
So in this case it should be a separate, follow-up pull request.